IdentityCore/src/controllers/broker/MSIDSSOExtensionInteractiveTokenRequestController.m

```diff
@@ -52,18 +53,53 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
     {
         MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Interactive broker extension flow finished. Result %@, error: %ld error domain: %@", _PII_NULLIFY(result), (long)error.code, error.domain);
         
+        if ([self shouldFallback:error])
+        {
+            MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Falling back to local controller.");
+            
+            [self.fallbackController acquireToken:completionBlock];
```

Comment:
should we return after this so that completionBlock is not executed?

Reply:
fixed